### PR TITLE
ENH: Add DatabaseInteractor and SPHARM-PDM dependency

### PR DESCRIPTION
**Pull request by allemangD**
Also add DatabaseInteractor and SPHARM-PDM links to the README.md table.

The module links should be added to the SlicerCMF module description.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -22,11 +22,11 @@ set(EXTENSION_DEPENDS
   Q3DC
   ShapePopulationViewer
   ShapeVariationAnalyzer
-  # SPHARM-PDM
+  DatabaseInteractor
+  SPHARM-PDM
```
Comment (jcfr) [inline CMakeLists.txt]: Considering listing these in alphabetical order

File: README.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ install these.
 | [Q3DC][gh-Q3DC]                                       | [cdash][cdash-Q3DC]                    |
 | [ShapePopulationViewer][gh-ShapePopulationViewer]     | [cdash][cdash-ShapePopulationViewer]   |
 | [ShapeVariationAnalyzer][gh-ShapeVariationAnalyzer]   | [cdash][cdash-ShapeVariationAnalyzer]  |
+| [DatabaseInteractor][gh-DatabaseInteractor]           | [cdash][cdash-DatabaseInteractor]      |
+| [SPHARM-PDM][gh-SPHARM-PDM]                           | [cdash][cdash-SPHARM-PDM]              |
```
Comment (jcfr) [inline README.md]: Considering listing these in alphabetical order

File: README.md [hunk 3]
```diff
@@ -40,6 +42,8 @@ install these.
 [gh-Q3DC]: https://github.com/DCBIA-OrthoLab/Q3DCExtension
 [gh-ShapePopulationViewer]: https://github.com/NIRALUser/ShapePopulationViewer
 [gh-ShapeVariationAnalyzer]: https://github.com/DCBIA-OrthoLab/ShapeVariationAnalyzer
+[gh-DatabaseInteractor]: https://github.com/DCBIA-OrthoLab/DatabaseInteractorExtension
```
Comment (jcfr) [inline README.md]: Considering listing these in alphabetical order

File: README.md [hunk 4]
```diff
@@ -52,4 +56,6 @@ install these.
 [cdash-Q3DC]: http://slicer.cdash.org/index.php?project=SlicerPreview&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=Q3DC
 [cdash-ShapePopulationViewer]: http://slicer.cdash.org/index.php?project=SlicerPreview&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=ShapePopulationViewer
 [cdash-ShapeVariationAnalyzer]: http://slicer.cdash.org/index.php?project=SlicerPreview&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=ShapeVariationAnalyzer
+[cdash-DatabaseInteractor]: http://slicer.cdash.org/index.php?project=SlicerPreview&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=DatabaseInteractor
```
Comment (jcfr) [inline README.md]: Considering listing these in alphabetical order

**Review by jcfr — APPROVED** [review]
Beside of listing in alphabetical order :+1:

**Comment by jcfr** [comment]
Also, it seems commits are associated with two different authors:

image

**Comment by allemangD** [comment]
> Also, it seems commits are associated with two different authors:
> 
> image

Ah... This is probably because some commits (editing README.md) were done in GitHub in browser, and others (editing CMakeLists.txt) were done locally, then I squashed them before creating the PR.

Alphebetizing the lists is a good idea. I'll do that and clean up the commit authors.

**Comment by jcfr** [comment]
> This is probably because some commits (editing README.md) were done in GitHub in browser, and others (editing CMakeLists.txt) were done locally, then I squashed them before creating the PR.

Look like the email associated with the commit is your personal one, is this one associated with your profile ?

**Review by jcfr — APPROVED** [review]

**Pull request merged**